mvcc/kvstore_txn.go

```diff
@@ -163,6 +167,7 @@ func (tr *storeTxnRead) rangeKeys(key, end []byte, curRev int64, ro RangeOptions
 			}
 		}
 	}
+	tr.trace.Step("Range keys from bolt db.")
```

Comment:
Can we remove all `."` at the end? It will be structure logged, so `.` doesn't seem necessary